expression/integration_test.go

```diff
@@ -2263,6 +2264,89 @@ func (s *testIntegrationSuite2) TestTimeBuiltin(c *C) {
 	result = tk.MustQuery(`select tidb_parse_tso(-1)`)
 	result.Check(testkit.Rows("<nil>"))
 
+	// for tidb_bounded_staleness
+	tk.MustExec("SET time_zone = '+00:00';")
```

Comment:
I suggest add following 2 testcases:

1. `select tidb_bounded_staleness(....), tidb_bounded_staleness(...)` and assert the timestamp is deterministic.
2. execute `select tidb_bounded_staleness` and assert the ts is different to ensure the cache have been removed.